webwhatsapi/objects/message.py

```diff
@@ -50,18 +54,26 @@ def __repr__(self):
 
 
 class MediaMessage(Message):
+    crypt_keys = {'document': '576861747341707020446f63756d656e74204b657973',
```

Comment:
Are these going to be constant for all clients?

Reply:
Yes, there is a key per media type, and another key for each message. I've got them from https://github.com/jlguardi/yowsup/blob/master/yowsup/layers/protocol_media/protocolentities/